src/prettytable/prettytable.py

```diff
@@ -2273,9 +2310,12 @@ def _get_formatted_html_string(self, options):
             for field in self._field_names:
                 if options["fields"] and field not in options["fields"]:
                     continue
+                if options["escape_header"]:
+                    field = escape(field)
```

Comment:
And without this.

Reply:
This does pass, because the method `_get_formatted_html_string` is not tested.
Should I add another 4 tests with the `result = t.get_html_string(escape_header=False,format=True)`?

Reply:
Thanks for the update! It's looking good, I'll try and give it a final review this weekend and we can hopefully merge it soon!